Array Representation 2: `numpydantic` (#2178)
linkml/linkml: docs/schemas/arrays.md

and many formats don't have a single canonical array type, generators may accommodate multiple array
representations.

> The basic representation supported by the pydantic generator is "List of lists" style array representation to be able to use
> arrays without any additional external dependencies beyond pydantic. 

Reviewer: ```suggestion
The basic representation supported by the pydantic generator is the "List of lists" style array representation. This can be used without any additional dependencies beyond pydantic.
```
